src/components/styles/styles.js

```diff
@@ -0,0 +1,45 @@
+/* Spacings for margenes and paddings */
+export const spacings = {
```

Comment:
I read it's recommended use multiple files, like `colors, spacings, fonts` because in the future can growing up and it's more difficult to maintain and less clear all in the same file

Reply:
I agree with @fcoronelmakingsense better keep them separated.